configs/katello/4.6.yaml

```diff
@@ -0,0 +1,142 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.6:
+    :redmine_version_id: 1561
+:prior_releases:
+  :4.5:
+    :redmine_version_id: 1543
+:code_name: Belgian Tripel
+:mash_scripts:
+  - katello-4.6-rhel7
+  - katello-4.6-thirdparty-candlepin-rhel7
+  - katello-4.6-el8
+  - katello-candlepin-4.6-el8
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.6
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/3.4
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 1.6.z
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 4.0.1
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.9.0
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.8
+    :repo: https://github.com/theforeman/foreman_virt_who_configure.git
+  :smart_proxy_container_gateway:
+    :branch: 1.0.6
+    :repo: https://github.com/Katello/smart_proxy_container_gateway.git
+  :katello-selinux:
+    :branch: 4.0.2
+    :repo: https://github.com/Katello/katello-selinux.git
+:ignores:
+:strict_keys: true
+:gpg_key: 2A6A4BAA
```

Comment:
Needs to be updated.

Reply:
Normally the release engineer (@pcreech this time) creates the key. There's no reason we can't generate the key earlier so we can move it up in the planning if that makes things easier in the future.

Reply:
Thanks @ewoud and @pcreech : Updated keys.